.github/workflows/typedoc.yml

```diff
@@ -0,0 +1,32 @@
+name: Deploy TypeDoc on GitHub pages
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+env:
+  NODE_VERSION: 16.x
+  ENTRY_FILE: 'src/index.ts'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
```

Comment:
These do not correspond to the latest template.

Reply:
Thanks.

Script taking this from cheminfo `.github` repository I think. 

Maybe we can update it there, and also here.

Reply:
Looks up to date [in cheminfo](https://github.com/cheminfo/.github/blob/main/workflow-templates/typedoc.yml#L16) too.